packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -172,6 +172,12 @@ class Directionality extends InheritedWidget {
 /// ```
 /// {@end-tool}
 ///
+/// Directly drawing an [Image] with opacity is faster than using [Opacity]
```

Comment:
I think this is an unnecessary specialization (on [Image]). Maybe make it more generic (and target people who can just fill with a color that's (0xnotFFXXXXXX) etc.

Reply:
Do you mean that any Widget with a `color` and `colorBlendMode` parameters could use this trick? Do you have an example Widget, or a piece of sample code, that I can add here?

If we're to make it more generalized, I think that it's a good idea to provide more pieces of sample code than just the `Image`.

Reply:
I just meant we can use this section to remind people to do

```dart
return Container(color: Color(0x7FFF0000);
```

instead of

```dart
return Opacity(
  opacity: 0.5,
  child: Container(color: Color(0xFFFF0000)),
);
```

or something

Reply:
How about the latest patch?